cmd/root_test.go

```diff
@@ -799,8 +790,9 @@ func TestGitWorktree(t *testing.T) {
 	as.NoError(err)
 
 	assertStats(t, as, statz, map[stats.Type]int32{
-		stats.Traversed: 60,
-		stats.Matched:   60,
+		stats.Traversed: 80,
+		stats.Matched:   80,
```

Comment:
I still don't understand why the stats have changed here.

Reply:
That helps, thanks. But I'm still confused about why these numbers changed from before. Did we change what the test is doing? Or did we change what the underlying code is doing?